flink-connector-kafka/src/main/java/org/apache/flink/connector/kafka/sink/KafkaSinkBuilder.java

```diff
@@ -61,7 +61,7 @@
 public class KafkaSinkBuilder<IN> {
 
     private static final Logger LOG = LoggerFactory.getLogger(KafkaSinkBuilder.class);
-    private static final Duration DEFAULT_KAFKA_TRANSACTION_TIMEOUT = Duration.ofHours(1);
+    private static final Duration DEFAULT_KAFKA_TRANSACTION_TIMEOUT = Duration.ofMinutes(15);
```

Comment:
It seems a reasonable change, but I am concerned about changing the default of an option. The cautious approach would be to introduce a new parameter with the new default, so the user can choose when to move to the new default and there are no unexpected changes in behaviour for minor upgrades.